src/controllers/article.js

```diff
@@ -0,0 +1,153 @@
+import Slug from 'slug';
+import ArticleModel from '../helpers/articles';
+
+/**
+  * @description class representing Article Controller
+  * @class ArticleHandler
```

Comment:
Did you mean to use `ArticleController`, or was the choice of a JSDoc `@class` name that is different from the actual class name intentional?